myapp/app/views/tasks/_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render 'errors', task: task %>
 
 <div class='container-sm rounded-3' style='padding:10px;'>
-  <%= form_with model: task do |form| %>
+  <%= form_with model: task, local: true do |form| %>
```

Comment:
[my note] I did not know this option. Without it, it would be XHR communication 👀 
https://api.rubyonrails.org/v7.0.4/classes/ActionView/Helpers/FormHelper.html#method-i-form_with